web/instances/instances.go

```diff
@@ -146,10 +146,6 @@ func listHandler(c echo.Context) error {
 
 	objs := make([]jsonapi.Object, len(is))
 	for i, in := range is {
-		in.OAuthSecret = nil
```

Comment:
Do not forget this API is only available through the admin socket. But I agree we can continue to filter the secrets until we have a valid usecase.